ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -0,0 +1,734 @@
+
+### Contents of the inventory.yml file
+
+The following describes the full `ansible-avd-examples/single-dc-l3ls/inventory.yml` file used to describe the topology shown above.
+
+It's important that the hostnames specified in the inventory exist either in DNS or in the hosts file on your Ansible host to allow successful name lookup and be able to reach the switches directly. To test this, you must be able to successfully ping the host, for example, `ping dc1-spine1` from your Ansible host. Alternatively, if there is no DNS available, or if devices need to be reached using a fully-qualified domain-name (FQDN) define `ansible_host` to be an IP address or FQDN for each device - see dc1-spine1/2 below for an example:
+
+```yml
+---
+all:
+  children:
+    FABRIC:
+      children:
+        DC1:
+          children:
+            DC1_SPINES:
+              hosts:
+                dc1-spine1:
+                  ansible_host: 172.16.1.11
+                dc1-spine2:
+                  ansible_host: 172.16.1.12
+```
+
+However, going forward we expect that all hostnames specified are reachable without this workaround, hence the entire inventory file looks as follows:
+
+```yml
+---
+all:
+  children:
+    FABRIC:
+      children:
+        DC1:
+          children:
+            DC1_SPINES:
+              hosts:
+                dc1-spine1:
+                dc1-spine2:
+            DC1_L3_LEAVES:
+              hosts:
+                dc1-leaf1a:
+                dc1-leaf1b:
+                dc1-leaf2a:
+                dc1-leaf2b:
+            DC1_L2_LEAVES:
+              hosts:
+                dc1-leaf1c:
+                dc1-leaf2c:
+
+        NETWORK_SERVICES:
+          children:
+            DC1_L3_LEAVES:
+            DC1_L2_LEAVES:
+        CONNECTED_ENDPOINTS:
+          children:
+            DC1_L3_LEAVES:
+            DC1_L2_LEAVES:
+```
+
+The `NETWORK_SERVICES` section does two things:
+
+1. It references the `ansible-avd-examples/single-dc-l3ls/NETWORK_SERVICES.yml` group variable file that specifies the VRFs and VLANs used in the entire fabric.
+2. Defines the scope of the fabric where these VRFs and VLANs are applied, in this case the two groups `DC1_L3_LEAVES` and `DC1_L2_LEAVES`
+
+The `CONNECTED_ENDPOINTS` section also does two things:
+
+1. It references the `ansible-avd-examples/single-dc-l3ls/CONNECTED_ENDPOINTS.yml` group variable file that specifies all endpoints, which would typically be servers.
+2. Defines the scope of where these endpoints are connected to the fabric, in this case the two groups `DC1_L3_LEAVES` and `DC1_L2_LEAVES`.
+
+## Defining device types
+
+Since we're building a L3LS network, we must tell Ansible AVD which devices are spines, L3 leaves, L2 leaves etc. We've already grouped the devices in the inventory and so in this example this is done using the group variable files with the following names and content:
```

Comment:
```suggestion
Since we're building a L3LS network, we must tell Ansible AVD which devices are spines, L3 leaves, L2 leaves etc. We have already grouped the devices in the inventory and so in this example this is done using the group variable files with the following names and content:
```